src/pages/Browse/index.tsx

```diff
@@ -34,7 +34,7 @@ import Loading from '../../components/Loading';
 import CodeView from '../../components/CodeView';
 import styles from './styles.module.scss';
 
-export type PublicProps = {};
+export type PublicProps = Record<string, unknown>;
```

Comment:
Is this another case where we don't need `PublicProps` at all?

Reply:
I don't think we should change that right now. That was working before and I don't think we complained about it.

Reply:
Fair enough. I am a bit concerned about the fact that I don't think that `Record<string, unknown>` is actually equivalent to `{}`. When we used `{}` what we were really saying is "There are no public props", which I don't think is the same thing as `Record<string, unknown>`. I think we're just doing a replacement to please the linter, but we're replacing it with something inaccurate.

I wonder if there's a different, valid, notation we can use in TS to say, "this is an empty type with no properties"?

Reply:
Yeah, that's part of the reasons why I don't like TS anymore.. I tried to use `{} as Record<...>` but because it's a type, we cannot do that.

Reply:
Ok, so I guess our strategy for now is to just keep TS happy with a minimum of effort and possibly migrate away from it some day?

Reply:
I don't really have a better alternative for now. Given that it compiles and the linter is happy, I guess we're good?

Reply:
One more thing about this `PublicProps` thing. I suggested we remove it (here and elsewhere) as it's not actually used anywhere in the file. I.e., TS doesn't enforce it. So all it's really doing is serving as some documentation to show that there are no public props. Documentation that could easily be missed or ignored by a developer, and not necessarily any more useful than recognizing that no `PublicProps` type means that no public props are allowed.

Reply:
> I don't really have a better alternative for now. Given that it compiles and the linter is happy, I guess we're good?

That's what my second-to-last comment was about. I guess we can just accept it for now. I agree with not spending too much time on nitty things for TS.